### ISSUE TITLE
PKGBUILD Immediately Fails When Used With: `paru -Ui` (`makepkg -si`)

### ISSUE THREAD

**Issue by ThisNekoGuy** [Frogging-Family/linux-tkg#377]
[Paru's](https://github.com/morganamilo/paru) "paru -Ui" command is essentially just `makepkg -si` by default but it refuses to run starting at the `plain` lines.

Commenting these out doesn't quite solve the problem either, because then it fails to understand the PKGBUILD a bit further down about something else. :/

(Wanted to use Paru for some convenience features is brings when using PKGBUILDs but I figured I'd put this here to make note of the issue and hopefully someone can get to the bottom of it.)

**Comment by gardotd426** [comment]
I'm not sure what convenience features you're wanting that `makepkg` can't already provide, but yeah with such a dynamic and complicated PKGBUILD I don't know if there's a way to make this work very easily. And TKG ain't getting rid of no Frog ASCII :frog: lol

**Comment by ThisNekoGuy** [comment]
@gardotd426 
> I'm not sure what convenience features you're wanting that makepkg can't already provide

Paru has it's own settings that can be configured for stuff; but I mainly use it for cleaning up PKGBUILD dependencies for me and its simple use of chroots :p

Moreso, maybe someone can find the issues within Paru and fix it over there; I didn't put down the issue over there because it would likely just get dismissed as a "non-issue" because it's a non-conventional PKGBUILD that isn't on the AUR either

**Comment by gardotd426** [comment]
Yeah this seems like an issue where paru just isn't compatible with this PKGBUILD, and the only option would be to take it up with the paru devs (though you likely will get the reaction you predicted lol).
